src/main/java/internet/shop/dao/impl/ProductDaoImpl.java

```diff
@@ -0,0 +1,45 @@
+package internet.shop.dao.impl;
+
+import internet.shop.dao.ProductDao;
+import internet.shop.db.Storage;
+import internet.shop.lib.Dao;
+import internet.shop.model.Product;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ProductDaoImpl implements ProductDao {
+    @Override
+    public Product createProduct(String name, Double price) {
+        return new Product(name, price);
+    }
```

Comment:
I think it`s not a storage logic to create the object

Reply:
You're right my anonimous friend :)